### PR TITLE
Integration test

### PR DESCRIPTION
**Pull request by kdzwinel**
Adds one simple integration test that visits multiple known websites and verifies collected data.

I confirmed that this test would have prevented https://github.com/duckduckgo/tracker-radar-collector/pull/24 :

failures

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -8,7 +8,7 @@
   "scripts": {
     "crawl": "node ./cli/crawl-cli",
     "test": "npm run lint && tsc && npm run unit",
-    "unit": "find . -name \"*.test.js\" -not -path \"./node_modules/*\" -exec node {} \\;",
+    "unit": "node ./tests/runTests.js",
```
Comment (kdzwinel) [inline package.json:11]: Some test failures didn't cause the whole testing to fail, so I had to write my own small runner to work around this. I guess, at some point, I should consider just using one of existing test runners.

File: package.json [hunk 2]
```diff
@@ -20,7 +20,7 @@
     "test"
   ],
   "engines": {
-    "node": ">=10.0"
+    "node": ">=10.17"
```
Comment (kdzwinel) [inline package.json:23]: `unhandled-rejections=strict` that allows us to throw when unhandled promise rejection happens is only available since 10.17

**Comment by jdorweiler** [comment]
👍 thanks for adding this

**Comment by loco22loco** [comment]
> Adds one simple integration test that visits multiple known websites and verifies collected data.
> 
> 
> 
> I confirmed that this test would have prevented https://github.com/duckduckgo/tracker-radar-collector/pull/24 :
> 
> 
> 
> failures
> 
>

**Comment by kdzwinel** [comment]
Thanks for double-checking @loco22loco!

**Pull request merged**